jena-arq/src/test/java/org/apache/jena/sparql/syntax/TestSSE_Builder.java

```diff
@@ -233,4 +249,42 @@ public void testBuildExpr_14() {
                       "(!= ?x ?y)") ;
     }
 
+    @Test
+    public void testBuildTable_01() {
+        Op expected = OpTable.unit();
+        Op actual = SSE.parseOp("(table unit)");
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void testBuildTable_02() {
+        Op expected = OpTable.empty();
+        Op actual = SSE.parseOp("(table empty)");
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void testBuildTable_03() {
+        Op expected = OpTable.create(TableFactory.create(Var.alloc("x"), NodeConst.nodeTrue));
+        Op actual = SSE.parseOp("(table (vars ?x) (row (?x true)))");
+        assertEquals(expected, actual);
+    }
+    
+    @Test
+    public void testBuildTable_04() {
+        // Can't test for equality because can't create a BNode in a way that equality will
+        // succeed because OpTable does strict equality and ignores NodeIsomorphismMap
+        SSE.parseOp("(table (vars ?x) (row (?x _:test)))");
+    }
+
+    @Test(expected = ExprBuildException.class)
```

Comment:
For some reason `expected` is not working as expected ;)

Reply:
Apparently it's because we extend `TestCase` so JUnit assumes a JUnit 3.8 test which doesn't support expected, adding `@RunWith(JUnit4.class)` to the test class resolves this

See https://stackoverflow.com/a/50867198/107591